cmd/regression-retrieval/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"os"
+
+	"github.com/src-d/regression-retrieval/prometheus"
+	"github.com/src-d/regression-retrieval/test"
+	"github.com/src-d/regression-retrieval/test/gitcollector"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/src-d/regression-core"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+var description = `data-retrieval utils regression tester.
+
+This tool executes several versions of data-retrieval utils and compares query times and resource usage. There should be at least two versions specified as arguments in the following way:
+
+* v0.12.1 - release name from github (https://github.com/src-d/your-data-retrieval-util/releases). The binary will be downloaded.
+* latest - latest release from github. The binary will be downloaded.
+* remote:master - any tag or branch from repository. The binary will be built automatically.
+* local:fix/some-bug - tag or branch from the repository in the current directory. The binary will be built.
+* local:HEAD - current state of the repository. Binary is built.
+* pull:266 - code from pull request #266 from your-data-retrieval-util repo. Binary is built.
+* /path/to/your-data-retrieval-util - a binary built locally.
+
+The repositories and downloaded/built binaries are cached by default in "repos" and "binaries" repositories from the current directory.
+`
+
+type Options struct {
+	regression.Config
+
+	CSV bool `long:"csv" description:"save csv files with last result"`
+
+	Kind string `long:"kind" description:"kind of utility that will be tested, currently only gitcollector is supported"`
+
+	// prometheus pushgateway related options
+	Prometheus bool `long:"prom" description:"store latest results to prometheus"`
+	PromConfig prometheus.PromConfig
+	CIConfig   prometheus.CIConfig
+}
+
+// TODO(@lwsanty) refactor to accept kind?
+func main() {
+	options := Options{
+		Config: regression.NewConfig(),
+	}
+
+	parser := flags.NewParser(&options, flags.Default)
+	parser.LongDescription = description
+
+	args, err := parser.Parse()
+	if err != nil {
+		if err, ok := err.(*flags.Error); ok {
+			if err.Type == flags.ErrHelp {
+				os.Exit(0)
+			}
+		}
+
+		log.Errorf(err, "Could not parse arguments")
+		os.Exit(1)
+	}
+
+	if options.Kind == "" {
```

Comment:
Setting a default in the option may be enough.

Reply:
Done.